openscenario/openscenario_interpreter/include/openscenario_interpreter/syntax/dynamic_constraints.hpp

```diff
@@ -52,12 +53,14 @@ struct DynamicConstraints
 
   const Double max_speed;
 
+  const Double max_jerk;
+
   explicit DynamicConstraints(
     const Double max_acceleration = Double::infinity(),
     const Double max_acceleration_rate = Double::infinity(),
     const Double max_deceleration = Double::infinity(),
     const Double max_deceleration_rate = Double::infinity(),
-    const Double max_speed = Double::infinity());
+    const Double max_speed = Double::infinity(), const Double max_jerk = Double::infinity());
```

Comment:
maxJerk field does not exist in [DynamicConstraints.](https://www.asam.net/static_downloads/ASAM_OpenSCENARIO_V1.2.0_Model_Documentation/modelDocumentation/content/DynamicConstraints.html)
Please remove it.